inst/extdata/gcam-china/unallocated_CStorage_province.csv

```diff
@@ -0,0 +1,38 @@
+# File: unallocated_CStorage_province.csv
+# Title: unallocated CO2 storage by province
+# Units: Mt
+# Description: Provincial unallocated CO2
```

Comment:
Source: PNNL's research based on Dahowski et al. (2010, 2017) and Yu et al. (2018)

Reply:
get